server/conn.go

```diff
@@ -37,50 +37,45 @@ type Conn struct {
 }
 
 var baseConnID uint32 = 10000
-
-// NewConn: create connection with default server settings
+var defaultServer *Server
+ // NewConn: create connection with default server settings
+//
+// Deprecated: Use [Server.NewConn] instead.
+// Deprecated: Use [Server.NewConn] instead.
```

Comment:
I'm unsure why GitHub messed up my suggestion.

There is no reason for this line to be duplicated

```suggestion
```